dev-python/pyperclip/pyperclip-1.6.0.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 python3_5 python3_6 pypy)
```

Comment:
Maybe you want to squash to python3_{4,5,6}, as you have done with the other packages?

Reply:
Yep, will squash them. I've pushed the commit. =)